### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by egirgin**
We have created the necessary docker files under the practice-app folder. All you need to run docker is installing and logging into docker. then go to practice-app and run `commands.sh` or run the commands inside it one by one. (1 `pip` command and 2 `docker-compose` commands). 

Please try and let us know whether it works or not.

PS: from now on, the server is running at the port 8080

### CODE REVIEW

File: practice-app/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
+
+EXPOSE 9876
+CMD [ "node", "backend/app.js" ]
```
Comment (zeritte) [inline practice-app/Dockerfile:19]: I propose `RUN npm start` would be more proper @egirgin

File: practice-app/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "practice-app-backend",
```
Comment (zeritte) [inline practice-app/package.json:2]: this should not have backend tag I think @egirgin

File: practice-app/Dockerfile (hunk 1 above)
Comment (egirgin) [reply practice-app/Dockerfile]: solved [here](https://github.com/bounswe/bounswe2020group4/pull/102/commits/96d16ebf00d949904727fc7ddf85452a6144c703)

File: practice-app/package.json (hunk 2 above)
Comment (egirgin) [reply practice-app/package.json]: solved [here](https://github.com/bounswe/bounswe2020group4/pull/102/commits/81bbe81d712c483f306a053da35d38058fa688c9)

**Pull request merged**